software/turtlebot4_desktop.md

```diff
@@ -0,0 +1,148 @@
+---
+sort: 5
+---
+
+# TurtleBot 4 Desktop
+
+The `turtlebot4_desktop` metapackage contains packages used for visualising and interfacing with the TurtleBot 4 from a PC.
+
+## Installation
+
+Source code is available [here](https://github.com/turtlebot/turtlebot4_desktop).
+
+```note
+The `turtlebot4_desktop` metapackage can be installed on a PC running Ubuntu Desktop 20.04 with ROS 2 Galactic.
+```
+
+### Debian installation
```

Comment:
Should this say something alone the lines of Debian package (Ubuntu) to avoid confusion with the Debian OS?